pkg/workspace/data_values_pre_processing.go

```diff
@@ -67,6 +67,8 @@ func (pp DataValuesPreProcessing) apply(files []*FileInLibrary) (*datavalues.Env
 		if len(typeCheck.Violations) > 0 {
 			return nil, nil, schema.NewSchemaError("One or more data values were invalid", typeCheck.Violations...)
 		}
+		//Walk updates node's validations meta from Node's assigned type
```

Comment:
nit: for consistency...

```suggestion
		// Walk updates node's validations meta from Node's assigned type
```